buildtap.py

```diff
@@ -420,11 +420,11 @@ def inf2cat(self, x64):
     def sign(self, file):
         certspec = ""
         if self.sign_cert:
-            certspec += "/f %s " % self.sign_cert
+            certspec += "/f '%s' " % self.sign_cert
             if self.cert_pw:
-                certspec += "/p %s " % self.cert_pw
+                certspec += "/p '%s' " % self.cert_pw
         else:
-            certspec += "/s my /n %s " % self.sign_cn
+            certspec += "/s my /n '%s' " % self.sign_cn
```

Comment:
I guess, build process should have failed right after "SignTool Error: No certificates were found that met all the given criteria".

If it failed, you would not have overlooked it.